docs/claim-transparency.md

```diff
@@ -60,27 +57,19 @@ This section describes the semantics of each field in the claim format:
 - **subject** _(array of objects, required)_:
   Set of artifacts (e.g., source code, or some binary) that the claim applies to.
   - **subject[*].digest** and **subject[*].name** as defined by Statement in the in-toto standard.
-- **issuer** _(object, required)_:
-  Identifies the entity (person, organization, or an automated tool) that reviewed/audited/examined
-  the artifact in the subject of this claim.
-  - **issuer.id** _(string ([TypeURI](https://github.com/in-toto/attestation/blob/main/spec/field_types.md#TypeURI)), required)_:
-    URI indicating the issuer’s identity (e.g., mailto:issuer@example.com).
 - **claimType** _(string ([TypeURI](https://github.com/in-toto/attestation/blob/main/spec/field_types.md#TypeURI)), required)_:
   URI indicating what type of claim was issued. It determines the meaning of claimSpec and evidence below.
-- **metadata** _(object, required)_:
-  Additional details about the claim.
-
-  - **metadata.issuedOn** _(string ([Timestamp](https://github.com/in-toto/attestation/blob/main/spec/field_types.md#Timestamp)), required)_:
-    The timestamp of when the assessment of the artifact was performed. In most cases, the
-    assessment involves manually auditing or reviewing some source code. The timestamp in this case
-    does not have to be the accurate moment the review process started, but only the date on which
-    the review was done.
-  - **metadata.validFrom** _(string ([Timestamp](https://github.com/in-toto/attestation/blob/main/spec/field_types.md#Timestamp)), required)_:
-    The timestamp from which the claim is effective, and the artifact is endorsed for use. In most
-    cases, it is the same as issuedOn timestamp, but can be any time after that.
-  - **metadata.expiresOn** _(string ([Timestamp](https://github.com/in-toto/attestation/blob/main/spec/field_types.md#Timestamp)), required)_:
+- **issuedOn** _(string ([Timestamp](https://github.com/in-toto/attestation/blob/main/spec/field_types.md#Timestamp)), required)_:
+    The timestamp at which this claims was generated.
+- **validity** _(object, required)_:
+  Validity duration of the claim.
+
+  - **validity.notBefore** _(string ([Timestamp](https://github.com/in-toto/attestation/blob/main/spec/field_types.md#Timestamp)), required)_:
+    The timestamp from which the claim is effective, and the artifact is endorsed for use. Must be
+    equal or after the issuedOn timestamp.
+  - **validity.notAfter** _(string ([Timestamp](https://github.com/in-toto/attestation/blob/main/spec/field_types.md#Timestamp)), required)_:
     The timestamp of when the artifact is no longer endorsed for use. This, combined with the
-    validFrom field, is particularly useful for implementing passive revocation.
+    `notBefore` field, is particularly useful for implementing passive revocation.
```

Comment:
You could move "This, combined with `notBefore` field, is particularly useful for implementing passive revocation." to up **validity**, i.e., "Validity duration of a claim enables implementing passive revocation."